app/models/chargeback_container_project.rb

```diff
@@ -0,0 +1,98 @@
+class ChargebackContainerProject < Chargeback
+  set_columns_hash(
+    :start_date           => :datetime,
+    :end_date             => :datetime,
+    :interval_name        => :string,
+    :display_range        => :string,
+    :project_name         => :string,
+    :cpu_used_cost        => :float,
+    :cpu_used_metric      => :float,
+    :cpu_cost             => :float,
+    :cpu_metric           => :float,
+    :fixed_compute_1_cost => :float,
+    :fixed_compute_2_cost => :float,
+    :fixed_2_cost         => :float,
+    :fixed_cost           => :float,
+    :memory_used_cost     => :float,
+    :memory_used_metric   => :float,
+    :memory_cost          => :float,
+    :memory_metric        => :float,
+    :net_io_used_cost     => :float,
+    :net_io_used_metric   => :float,
+    :net_io_cost          => :float,
+    :net_io_metric        => :float,
+    :total_cost           => :float
+  )
+
+  def self.build_results_for_report_ChargebackContainerProject(options)
+    # Options:
+    #   :rpt_type => chargeback
+    #   :interval => daily | weekly | monthly
+    #   :start_time
+    #   :end_time
+    #   :end_interval_offset
+    #   :interval_size
+    #   :owner => <userid>
+    #   :tag => /managed/environment/prod (Mutually exclusive with :user)
+    #   :chargeback_type => detail | summary
+    #   :entity_id => 1/2/3.../all rails id of entity
+
+    # Find Project by id or get all projects
+    id = options[:entity_id]
+    raise "must provide option :entity_id" if id.nil?
+
+    @groups = if id == "all"
+               ContainerGroup.all
+             else
+               ContainerGroup.where('container_project_id = ? or old_container_project_id = ?', id, id)
+             end
+
```

Comment:
> You can check the errors with bundle exec rubocop before push.

Thanks, I didnt know that was available.
This doesnt work though, something about a bad ```.rubocop.yml``` file